sections/semantics-embedded-content.include

```diff
@@ -5685,6 +5685,18 @@ zero or more <{track}> elements, then
   should not show this content to the user; it is intended for older Web browsers which do
   not support <{audio}>, so that legacy audio plugins can be tried, or to show text to the
   users of these older browsers informing them of how to access the audio contents.
+  
+  <div class="example">
+    In this example, a browser which does not support inline audio can present a download link to
+    the user.
+    
+    <xmp highlight="html">
+      <audio src="music.ogg" controls>
+        Your browser does not support the audio element.
+        <a href="music.ogg">Download the music file instead</a>.
```

Comment:
thoughts on adding the download attribute to this link?  i like the idea of giving that some more exposure.